web/draw_layer_builder.css

```
@@ -56,22 +56,52 @@
       fill-rule: evenodd;
       fill: none;
 
-      &.hovered {
-        stroke: var(--hover-outline-color);
-        stroke-width: var(--outline-width);
+      &:not(.free) {
+        &.hovered:not(.selected) {
+          stroke: var(--hover-outline-color);
+          stroke-width: var(--outline-width);
+        }
+
+        &.selected {
+          .mainOutline {
+            stroke: var(--outline-around-color);
+            stroke-width: calc(
+              var(--outline-width) + 2 * var(--outline-around-width)
+            );
+          }
+
+          .secondaryOutline {
+            stroke: var(--outline-color);
+            stroke-width: var(--outline-width);
+          }
+        }
       }
 
-      &.selected {
-        .mainOutline {
-          stroke: var(--outline-around-color);
-          stroke-width: calc(
-            var(--outline-width) + 2 * var(--outline-around-width)
-          );
+      &.free {
+        /*
+          When drawing the outline we use a mask in order to remove the parts
+          that are inside the shape. Unfortunately, this removes the part of the
+          outline that is inside the shape. To "fix" this we draw the outline
+          we increase the width to have what we want to be visible outside the
+          shape. This is not a perfect solution, but it works well enough.
```

The middle of this comment reads a little strangely, did you mean something like the following?
```suggestion
          When drawing the outline we use a mask in order to remove the parts
          that are inside the shape. Unfortunately, this removes the part of the
          outline that is inside the shape. To "fix" this we increase the width
          to have what we want to be visible outside the shape.
          This is not a perfect solution, but it works well enough.
```